.evergreen/stop-orchestration.sh

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 set -o errexit  # Exit the script with error if any of the commands fail
 
+# Fix orchestration path.
+if [[ "$(uname -s)" == CYGWIN* ]]; then
+  MONGO_ORCHESTRATION_HOME=$(cygpath -m $MONGO_ORCHESTRATION_HOME)
+fi
```

Comment:
Confirming, what prompted these additions of `cygpath -m $MONGO_ORCHESTRATION_HOME` to become necessary? (I don't believe they were necessary prior to this PR) Can the root cause be identified and reverted instead of relying on calls to `cygpath`?

Reply:
The idea was to move the logic out of EVG config.  My follow up idea was for drivers to write a config env file, and then we normalize that file.   For now, we can move the logic back into EVG config.

Reply:
Hmm, actually the issue was that "OSTYPE" was not defined on Windows in `shell.exec`, even though the shell was set to `bash`.  So I had to change that to use `uname` as well.